tests/license/license_test.go

```diff
@@ -577,6 +577,70 @@ var _ = Describe("{DeleteSecretRebootAllNodes}", func() {
 	})
 })
 
+// This test performs basic test of starting an application and destroying it (along with storage)
+var _ = Describe("{DisableCallHomeTest}", func() {
+	var contexts []*scheduler.Context
+	It("has to setup, validate and teardown apps", func() {
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("setupteardown-license-callhome-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+
+		currNode := node.GetNodes()[0]
+		Step(fmt.Sprintf("Set License expiry timeout to 1 hour"), func() {
+			err := Inst().V.SetClusterOpts(currNode, map[string]string{
```

Comment:
Do we need to restart PX service after setting Cluster options?

Reply:
i don't believe so?

Reply:
i'll add a reboot just in case